src/main/java/com/dnanexus/DXApplet.java

```diff
@@ -249,11 +249,22 @@ public Builder setTitle(String title) {
         private RunSpecification runSpec;
         @JsonProperty
         private String dxapi;
+        @JsonProperty
```

Comment:
we will let ohad decide if he allows changes to be made on the dnanexus toolkit level.